src/agenda/index.js

```diff
@@ -429,7 +429,7 @@ export default class AgendaView extends Component {
           overScrollMode='never'
           showsHorizontalScrollIndicator={false}
           showsVerticalScrollIndicator={false}
-          style={scrollPadStyle}
+          style={[scrollPadStyle, this.styles.scrollPadStyle]}
```

Comment:
> Full disclosure, it's been a really long time since I made this and I'm not on the project that uses this anymore. I'm gonna explain as best as I can what I did.

From what I can remember... As per defined in the class constructor, `this.styles.*` comes from `styleConstructor(props.theme)`. This means that `scrollPadStyle` should be passed in the `theme` prop. Extending the example in the README:

```jsx
<Calendar
  theme={{
    // pass in your custom styles
    scrollPadStyle: {
      ...
    }
  }}
/>
```

Reply:
Aren't you suppose to access it using `appStyle` (instead of `this.styles`?